.travis.yml

```diff
@@ -10,6 +10,9 @@ branches:
 before_install:
 - npm config set spin false
 - npm install -g bower coveralls pr-bumper
+- bower --version
+- npm install phantomjs-prebuilt
```

Comment:
For that I do apologize, it should have been documented much more clearly that his addon was never intended to support phantomjs.